examples/circuitplaygrounc_neopixel_0_1.py

```diff
@@ -0,0 +1,8 @@
+"""This example lights up the first and second NeoPixel, red and blue respectively."""
```

Comment:
This example appears to have a typo in the name.

Reply:
Yep already caught it. Thanks!